package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "bookstore-react",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
- [ ] [REQUIRED] It will be nice if add `redux`  and` prop-types` to `package.json` [screenshot from Pathwright - redux and react-redux](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone1/AddReduxAndReact-Redux.png), [typechecking props](https://reactjs.org/docs/typechecking-with-proptypes.html) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/section/propTypes.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896806/))